src/layouts/ToastNotifications/ToastNotifications.test.jsx

```diff
@@ -35,7 +36,7 @@ describe('ToastNotifications', () => {
 
   describe('when rendered', () => {
     beforeEach(() => {
-      jest.useFakeTimers()
+      vi.useFakeTimers()
```

Comment:
very nice that there's a plug and play equivalent to the timers

Reply:
it's amazing how well they've done it